updates/version.yaml

```
@@ -68,3 +68,6 @@
 1.4.7: Fixes redirect bug in Account component / Update translations and separate user and group management.
 1.4.8: Fixes a bug where calling MailBlocker::removeBlock could remove all mail blocks for the user.
 1.5.0: !!! Required password length is now a minimum of 8 characters. Previous passwords will not be affected until the next password change.
+1.5.1:
+    - User IP addresses are now logged. Introduce registration throttle.
```

@daftspunk There's a couple of other changes being released with 1.5.1 - the main one being a "remember login" choice. (https://github.com/rainlab/user-plugin/pull/312)
```suggestion
    - User IP addresses are now logged. Introduce registration throttle and "remember login" functionality.
```